ajax/libs/bootstrap-calendar/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "bootstrap-calendar",
+  "version": "0.2.5",
+  "description": "Bootstrap full view calendar",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Serhioromano/bootstrap-calendar.git"
+  },
+  "keywords": [
+    "calendar"
+  ],
+  "author": "Sergey Romanov",
+  "license": "MIT",
+  "filename": "js/calendar.min.js",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/Serhioromano/bootstrap-calendar.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "js/calendar*",
+          "js/language/*",
+          "css/*",
```

Comment:
[Line24](https://github.com/cdnjs/cdnjs/pull/10445/files#diff-0e9a5cd2f24939ff230bb01b2a499c44R) in `package.json` can combine with [line22](https://github.com/cdnjs/cdnjs/pull/10445/files#diff-0e9a5cd2f24939ff230bb01b2a499c44R22).